kotlin-jvm/settings.gradle.kts

```diff
@@ -1,5 +1,22 @@
 import de.fayard.refreshVersions.bootstrapRefreshVersions
 
+pluginManagement {
+    repositories {
+        mavenLocal()
+        mavenCentral()
+        jcenter()
+        google()
+        gradlePluginPortal()
+    }
+    resolutionStrategy {
+        eachPlugin {
+            if (requested.id.id == "com.squareup.sqldelight") {
+                useModule("com.squareup.sqldelight:gradle-plugin:1.4.3")
+            }
+        }
+    }
+}
```

Comment:
This prevents refreshVersions from managing the version of the Gradle plugin.

Please use `buildscript` setup instead, with the `Square.sqlDelight.gradlePlugin` dependency constant.

Reply:
Hey
I had tried that method at first after I read up on refreshVersions, but gradle keeps throwing 
`Plugin [id: 'com.squareup.sqldelight'] was not found in any of the following sources:` when trying to apply the plugin in `build.gradle.kts` 
(tried both` apply(plugin="com.squareup.sqldelight")` as well as `id("com.squareup.sqldelight")` )

That's why I decided to use this. Let me know if there's a better way of doing so :)